src/index.html

```diff
@@ -1,14 +1,63 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+
   </head>
+
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img src="/src/images/logo.png"
+          alt="moyo"
+          class="logo-img"
+        >
```

Comment:
```suggestion
        <img
          src="/src/images/logo.png"
          alt="moyo"
          class="header__img"
        >
```

Reply:
<img width="621" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/a9d17638-40a1-484e-8cea-3bb7483b08f9">
still not fixed